x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/LocalPhysicalPlanOptimizerTests.java

```diff
@@ -93,13 +90,7 @@ public class LocalPhysicalPlanOptimizerTests extends MapperServiceTestCase {
      */
     private static final int KEYWORD_EST = EstimatesRowSize.estimateSize(DataTypes.KEYWORD);
 
-    private EsqlParser parser;
-    private Analyzer analyzer;
-    private LogicalPlanOptimizer logicalOptimizer;
-    private PhysicalPlanOptimizer physicalPlanOptimizer;
-    private EsqlFunctionRegistry functionRegistry;
-    private Mapper mapper;
-
+    private TestPlannerOptimizer plannerOptimizer;
```

Comment:
Moved all of this, including the various `plan` methods, into `TestPlannerOptimizer` - which is also used for the query translator spec tests.